### PR TITLE
Applied most style changes from the new figma design

### PR DESCRIPTION
**Pull request by Mink-Mink**
You can see the figma stylesheet here https://www.figma.com/file/q7DDvvkSKfYpT0yheLexht/AI-Safety-ccc-ccstan99

- Most of the changes are there
- I didn't add a gray "tree" structure to the collapsable question sections because that seemed more complex
- Also didn't move search bar into the header because it's unclear to me how that would interact with the tags page (do we put tag filtering there? Do we make logic so that if someone types a question in there they get switched back to the base page?)
- Added a back button to the tags page - it wasn't in the design, but needing to edit the URL or click on stampy icon is really unintuitive when wanting to go back to the base question page without picking a tag
- See comments below for screenshots of the new look
- As you can see in the dialogue post, currently it highlights only a part of the row; this is down to how the form there is structured, but it seemed like a bit too much of a refactor for me to attempt right away.
- Blank space, in case I remember something later.

Note from @Aprillion: Related to issue #285

### CODE REVIEW

**Comment by Mink-Mink** [comment]
image

**Comment by Mink-Mink** [comment]
image

**Comment by Mink-Mink** [comment]
image

**Comment by Mink-Mink** [comment]
image

**Comment by Mink-Mink** [comment]
image

**Comment by Mink-Mink** [comment]
image

**Comment by Mink-Mink** [comment]
image

**Comment by Mink-Mink** [comment]
image

**Comment by Mink-Mink** [comment]
image

**Comment by Mink-Mink** [comment]
> image

Black border here is due to the screenshotting software, it's clean.

**Comment by Mink-Mink** [comment]
I can't seem to edit the original post, but some things I forgot to mention are:
- As you can see in the dialogue post, currently it highlights only a  part of the row; this is down to how the form there is structured, but it seemed like a bit too much of a refactor for me to attempt right away.
- Blank space, in case I remember something later.

File: app/root.css [hunk 1]
```diff
@@ -8,32 +8,36 @@
...
   --borderColorButton: #999;
-  --bgColorInput: #fff;
-  --bgColorButton: #ddd;
+  --bgColorButton: #dcdcdc;
   --bgColorHighlight: #c0d3e4;
   --bgColorTableRows: #e8f7fe;
   --colorLink: #79f;
   --colorLinkVisited: #8ac;
-  --colorTooltip: #d1d1d1;
+  --colorTooltip: rgba(48, 115, 183, 0.1);
```
Comment (Aprillion) [inline app/root.css:26]: * [ ] could you please find a better background for `.answer .link-popup`? transparent `--colorTooltip` is no good for that:
  image

@ccstan99 - I didn't find the popup styles in Figma - I'm happy if @Mink-Mink will pick from existing bg colors unless you have some new color in mind and just forgot to specify it yet

File: app/components/icons-generated/Undo.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+const SvgUndo = (props) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    height={24}
+    viewBox="0 -960 960 960"
+    width={24}
+    {...props}
+  >
+    <path
+      className="tag_svg__gray"
```
Comment (Aprillion) [inline app/components/icons-generated/Undo.js]: good try 👍 but this needs to be set in the SVG file itself as `class="gray"`, not a manual change in generated file

I pushed a fix commit to your forked repo, please pull before making more changes, thanks!

**Review by Aprillion — CHANGES_REQUESTED** [review]

File: app/root.css (hunk 1 above)
Comment (ccstan99) [reply app/root.css:26]: No transparency please. No new color. Feel free to pick from the existing bg color.

**Comment by ccstan99** [comment]
A few other minor things:

- "Show me even more (more) questions about AI Safety..." background should be transparent not darker gray.
(Please fix typo for us: deduplicate "more more")
- Question titles should be bolded to contrast from answers.
- All tags border outline should be blue (same color as text) not gray.

If they can be easily incorporated, please also match Figma for
- Size of tags text on tags page
- "None of these: Request an answer to my exact question above" should be styled
- Title of popup dialogues should be larger & bold
- Please use Figma specified colors for background & foreground. 
Screenshot 2023-09-30 at 4 51 51 PM
Screenshot 2023-09-30 at 4 52 01 PM

**Comment by ccstan99** [comment]
Also, you were referring to a slightly out-of-date Figma file. Please use [this for reference](https://www.figma.com/file/ipifwmO1KYfqakpl3srFIL/2023-08-08-AI-Safety-ccc-ccstan99?type=design&node-id=90%3A2047&mode=design&t=o2ArxfDRusbkWq7n-1). Thanks!

**Review by Aprillion — APPROVED** [review]
adding a few more color changes to match the Figma designs from #285 (only where I noticed some differences while I was trying to find color for the popups, now reusing `--bgColorHighlight` for both hover in search results and popups)

image

approving from code review perspective, we can merge if designers won't have any more major objections (minor fixes can be done later if we won't notice them right now)

deployed to https://stampy-ui.aprillion.workers.dev/

**Review by ccstan99 — APPROVED** [review]
This is much improved, thank you! I'm happy for this to be approved & merged but let’s keep the original issue with link to Figma file open. I’ll add any items still outstanding to that issue.

**Comment by ccstan99** [comment]
One last glaring thing if we can sneak it into this PR... Can we please remove the border outline around "Show me even more questions about AI Safety..."? The highlight on hover is fine but there's no need to draw that much attention to it otherwise.

**Comment by Aprillion** [comment]
border removed, merging now and let's solve follow-up issues in future PRs
image

**Review by Aprillion — APPROVED** [review]

**Pull request merged**